### ISSUE TITLE
Use badges to display status in README

### ISSUE THREAD

**Issue by alanyee** [os-autoinst/qa-tools-backlog-assistant#22]
Use [badges](https://github.com/badges/shields) to display status in README rather than link to external website

**Comment by perlpunk** [comment]
yep, written here already: https://github.com/os-autoinst/qa-tools-backlog-assistant/pull/15#issuecomment-974036788
